src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsBtfBlocking.java

```diff
@@ -17,15 +17,15 @@
   private static final Dimension TABLET_PAGE_SIZE = new Dimension(850, 600);
   private static final Dimension MOBILE_SIZE = new Dimension(414, 736);
   private static final String ARTICLE_MIDDLE_SECTION_SELECTOR = "#ArticleMidSection.mw-headline";
+  private static final Page PAGE_DISABLE_BTF = new Page("project43", "SyntheticTests/Disable_BTF");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "project43" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)